src/prometheus-net.DotNetRuntime/EventListening/Parsers/RuntimeEventParser.cs

```diff
@@ -69,9 +69,9 @@ public static class Events
         {
             public interface CountersV3_0 : ICounterEvents
             {
-                event Action<MeanCounterValue> ThreadPoolThreadCount;
+                event Action<MeanCounterValue> ConnectionCount;
```

Comment:
You've definitely broken the tests with this change- each event source should have it's own class defined. You may be able to group all the ASP.NET counters together as I guess users would commonly want to enable them all simultaneously.